src/dev-app/cdk-experimental-menu/cdk-menu-demo.html

```diff
@@ -151,3 +151,42 @@ <h4>Custom Context Menu Position (Centered on Cursor)</h4>
     </div>
   </ng-template>
 </div>
+
+<div class="example-menu-container">
+  <h3>Radio items</h3>
+  <button cdkMenuItem [cdkMenuTriggerFor]="sizeMenu">Size menu (items directly in menu)</button>
+  <button cdkMenuItem [cdkMenuTriggerFor]="colorMenu">Color menu (items in menu group)</button>
+  <button cdkMenuItem [cdkMenuTriggerFor]="sizeAndColorMenu">Size & Color menu (items in directly in menu & in menu group)</button>
+
+  <ng-template #sizeMenu>
+    <div class="example-menu" cdkMenu (change)="onSizeChange($event)">
+      <button [checked]="size === 'Small'" cdkMenuItemRadio>Small</button>
+      <button [checked]="size === 'Normal'" cdkMenuItemRadio>Normal</button>
```

Comment:
Not changed in this PR, but shouldn't the `checked` be called something like `cdkMenuItemRadioChecked`?

Reply:
Yeah I want to overhaul the checkbox / radio stuff, but I'm going to wait to do that in another PR